.test-env

```diff
@@ -1,6 +1,6 @@
 # Configs for testing repo download:
 SDK_TESTING_URL="https://github.com/algorand/algorand-sdk-testing"
-SDK_TESTING_BRANCH="master"
+SDK_TESTING_BRANCH="app-txn-decode"
```

Comment:
Reminder to change this back once https://github.com/algorand/algorand-sdk-testing/pull/248 is merged
```suggestion
SDK_TESTING_BRANCH="master"
```